capstone/src/main/java/com/google/sps/data/VotingEvent.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.data;
+
+public class VotingEvent {
+  String date;
+  String location;
+  String name;
+
+  public void VotingEventSummary(String name, String date) {
```

Comment:
nit: Is this a constructor? Or just is this method only used for modifying name and date fields?

Reply:
It honestly depends on how Silas wants to use it.